roles/wordpress-install/tasks/main.yml

```diff
@@ -37,6 +37,17 @@
   when: item.value.site_install == True and (item.value.multisite.enabled | default(False) == False)
   changed_when: "'WordPress is already installed.' not in wp_install_results.stdout"
 
+- name: Setup Permalink Structure
+  command: wp rewrite structure {{ item.value.permalink_structure }}
+           --allow-root
+           --url="{{ item.value.env.wp_home }}"
+  args:
+    chdir: "{{ www_root }}/{{ item.key }}/current/"
+  register: wp_install_results
```

Comment:
You re-used the variable name from above but this should probably be more descriptive for this task. Like `wp_permalink_results`